apps/api/src/routes/applicationRouter.ts

```diff
@@ -408,6 +463,88 @@ applicationRouter.post(
 				});
 			}
 		},
-	),
+
+// Endpoint for reps to request revisions
+applicationRouter.post(
+	'/dac/request-revisions',
+	jsonParser,
+	withBodySchemaValidation(applicationRevisionRequestSchema, apiZodErrorMapping, async (req, res) => {
+		const { applicationId, revisionData, role } = req.body;
```

Comment:
Why did we move the applicationId from the path into the body? The feedback before had examples like:

Applicant submit: `application/{id}/submit`
DAC request revision: `application/{id}/dac/request-revision`
Rep request revision: `application/{id}/rep/request-revision`

Its consistent with existing endpoints, and is consistent with the REST pattern.